src/test/java/api/loans/CheckInByBarcodeTests.java

```diff
@@ -1369,6 +1369,89 @@ void overdueFineCalculatedCorrectlyWhenHourlyFeeFinePolicyIsApplied() {
       homeLocation.getJson().getString("name"), ownerId, feeFineId, 7.0));
   }
 
+  @Test
+  void overdueRecallFineCalculatedCorrectlyWhenRecallRequestCreation() {
```

Comment:
I think it's either When...Created or On...Creation

Reply:
thanks